fable-test-app/src/components/SubmitHoliday/Success.tsx

```diff
@@ -1,23 +1,22 @@
-import { forwardRef, useEffect } from 'react';
+import { useEffect } from 'react';
 
 import { Button } from '../../components';
 
-const Success = forwardRef<HTMLGcdsStepperElement>((_, heading) => {
+const Success: React.FC = () => {
 
   useEffect(() => {
-    if(heading && typeof heading !== "function" && heading.current) {
-      setTimeout(() => {
-        heading.current?.focus();
-      }, 50);
-    }
+
+    setTimeout(() => {
+      // document.querySelector('gcds-notice')?.focus();
+    }, 50);
```

Comment:
Do we still need this?

Reply:
We will when we have published the `gcds-notice` component. Just put it there to allow easy implementation when we have it.